### PR TITLE
[yaireo__tagify] update library changes up to version 4.23.0

### PR DESCRIPTION
**Pull request by Brakebein**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/yairEO/tagify/compare/v4.18.3...v4.23.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

Please have a closer look at the React stuff as I'm not an expert on this one.

### CODE REVIEW

**Review by Anonymous4078 — APPROVED** [review]

**Comment by MendleM** [comment]
@Anonymous4078 Please review this asap. It's blocking our project atm

**Comment by Brakebein** [comment]
@blutorange May you have a quick look on it?

File: types/yaireo__tagify/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/yaireo__tagify",
-    "version": "4.18.9999",
+    "version": "4.24.9999",
```
Comment (rbuckton) [inline types/yaireo__tagify/package.json:4]: 4.24 or 4.23? The PR title and description seem to indicate 4.23.

Comment (Brakebein) [reply types/yaireo__tagify/package.json:4]: While I was waiting for a review, a new version has been released including a little update. That's why I updated this pull request (didn't change the title though). In the meantime, there are actually two new minor releases.

**Review by Anonymous4078 — APPROVED** [review]

**Comment by Brakebein** [comment]
Ready to merge

**Pull request merged**